quinn-h3/src/qpack/block.rs

```diff
@@ -0,0 +1,365 @@
+use bytes::{Buf, BufMut};
+
+use super::prefix_int;
+use super::prefix_string;
+use super::ParseError;
+
+pub(super) enum HeaderBlocField {
```

Comment:
Is it really necessary to use `pub(super)` in this module, given that the module is not itself reexported?